bundles/org.openhab.binding.systeminfo/README.md

```diff
@@ -104,6 +104,7 @@ The binding introduces the following channels:
 
 | Channel ID         | Channel Description                                              | Supported item type | Default priority | Advanced |
 |--------------------|------------------------------------------------------------------|---------------------|------------------|----------|
+| load               | CPU Load in %                                                    | Number:Dimensionless| High             | False    |
```

Comment:
See my other comments, I am not sure that `Number:Dimensionless` is more appropriate than `Number`.